packages/asl-internal-ui/pages/enforcement/list/formatters/index.jsx

```diff
@@ -55,7 +55,9 @@ export default {
   },
   date: {
     format: value => {
-      return <span className="date">{formatDate(value, dateFormat.medium)}</span>;
+      return <span className="date">{
+        isValid(toDate(value)) ? format(value, dateFormat.medium) : '-'
```

Comment:
I've tried to use isValid before and had issues with false negatives, but using toDate looks like it will fix that issue 🎉.

Can you move this to replace https://github.com/UKHomeOffice/aspel-workspace/blob/main/packages/asl-components/src/utils.js#L45-L51 to standardise how we format dates safely rather than doing this in multiple places?